Tasks/day.js

```diff
@@ -1,15 +1,14 @@
 // Get day number
+'use strict'
+const DAYS = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
```

Comment:
```suggestion
const DAYS = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat',];
```
Better to use ',' after the last element in the array